doc/style-guide.md

```diff
@@ -44,3 +45,52 @@ quaility.
     character limit.
 
 **[top](#table-of-contents)**
+
+## Expressions
+
+  - Expressions in return statements should be surrounded by parenthesis for clarity.
```

Comment:
That is the note below about expressions containing one operand.  Is that not sufficient?